docs/core/integration-quality-scale/rules/iqs008.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "IQS008 - If it's a polling integration, set an appropriate polling interval"
+---
+
+IQS008 is as follows:
+
+If it's a polling integration, set an appropriate polling interval
+
+## Reasoning
+
+In an ideal world, all integrations would have a data interface that is push based, where the device or service lets us know when there is new data.
+This would decrease the amount of requests Home Assistant would make.
```

Comment:
```suggestion
In an ideal world, all integrations would have a push-based data interface, where the device or service would let us know when new data is available.
This would decrease the amount of requests Home Assistant would make.
```